mava/components/jax/training/advantage_estimation.py

```diff
@@ -27,6 +27,7 @@
 from mava.components.jax.training.base import Utility
 from mava.core_jax import SystemTrainer
 
+from mava.utils.jax_training_utils import normalize, denormalize
 
 @dataclass
 class GAEConfig:
```

Comment:
Looks good. We recently updated our code to use SimpleNamespaces instead. So you can maybe remove all the empty dataclasses and just use a SimpleNamespace.

Reply:
I followed exactly the same approach as in the GAE function in the current main branch.

Reply:
See [here](https://github.com/instadeepai/Mava/blob/develop/mava/components/jax/executing/action_selection.py#L39) for an example.